old_src/Sand/erc20/ERC20BaseToken.sol

```diff
@@ -2,8 +2,6 @@ pragma solidity 0.5.9;
 
```

Comment:
better to leave old_src alone
in latest hardhat branch we actually get rid of old_src